tests/static/locators.html

```diff
@@ -8,16 +8,22 @@
 <body>
     <a id="link" href="#" onclick="event.preventDefault()">Click</a>
     <a href="#" onclick="event.preventDefault()">Click</a>
+    <input id="input" type="checkbox" />
+    <input type="checkbox" />
     <script>
         window.result = false;
         window.dblclick = false;
+        window.check = false;
 
         document.querySelector('#link').addEventListener(
             'click', e => { result = true; }, false
         );
         document.querySelector('#link').addEventListener(
             'dblclick', e => { dblclick = true; }, false
         );
+        document.querySelector('#input').addEventListener(
+            'change', e => { window.check = event.currentTarget.checked; }, false
```

Comment:
How come `event` works here? Shouldn't it be `e`?

Reply:
Good catch! It works, but to not create confusion in the future, I changed it to `e`: https://github.com/grafana/xk6-browser/pull/362/commits/b1aece296a1db114735651d14dc93b7ccb1122c6